src/server/console/linux_virtual_terminal.cpp

```diff
@@ -655,7 +655,7 @@ std::future<std::unique_ptr<mir::Device>> mir::LinuxVirtualTerminal::acquire_dev
         {
             using namespace boost::iostreams;
             char line_buffer[1024];
-            stream<file_descriptor_source> uevent{fd, file_descriptor_flags::never_close_handle};
+            stream<file_descriptor_source> uevent{(int)fd, file_descriptor_flags::never_close_handle};
```

Comment:
`Fd` has an implicit conversion to `int`, so I'm surprised this is needed. But it should not be a c-style cast.